packages/hawtio/src/plugins/jmx/JmxTreeView.tsx

```diff
@@ -6,23 +6,54 @@ import { MBeanTreeContext } from './context'
 import './JmxTreeView.css'
 
 export const JmxTreeView: React.FunctionComponent = () => {
-  const { tree, setNode } = useContext(MBeanTreeContext)
-  const [expanded, setExpanded] = useState(false)
+  const { tree, node, setNode } = useContext(MBeanTreeContext);
+  const [expanded, setExpanded] = useState(false);
+  const [filteredTree, updateTree] = useState(tree.getTree());
```

Comment:
Normally, with React `useState` hook a state is accompanied with its `set` function, so instead of naming it `updateTree` it should be `setFilteredTree` according to the convention:
https://beta.reactjs.org/reference/react/useState#setstate